digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ApiUtils.scala

```diff
@@ -81,9 +76,8 @@ object ApiUtils {
   }
 
   def newQuery[T](workId: String, queryId: String, path: String, f: Params => T, params: Params, responseType: String): Any = {
-    val ret = Future { f(params) }
     Future { // Complete query and save results to s3 in future
-      val finished = Await.result(ret, Duration.Inf)
+      val finished = f(params)
```

Comment:
Jäin miettimään... Onks olemassa joku sellanen tilanne, jossa toi kysely jää johonkin jumiin, eikä oo palannu vielä vaikka kolmen vuorokauden päästäkään?
Eli vähän niinku, että pitäiskö täällä (tai jossain kerroksessa) olla kuitenkin vielä joku varmistus-maksimi-aikaraja, jonka jälkeen killataan kyselyt ja kaikki, jos ei vieläkään oo palannu?
Vai onks tuol jossain vielä se Digiroadin toteutuksesta peritty redirektausten maksimi, joka hoitaa ton sit?

Reply:
Kröhöm...
Joo ei mitään, tossa yllähän se.

" if (currentRetry <= MAX_RETRIES)   --- "